conans/test/functional/command/install/install_update_test.py

```diff
@@ -316,3 +316,20 @@ def test_fail_usefully_when_failing_retrieving_package(self):
         # Try to install ref2, it will try to download the binary for ref1
         client.run("install {}".format(ref2), assert_error=True)
         self.assertIn("ERROR: Error downloading binary package: '{}'".format(pref1), client.out)
+
+    def test_evil_insertions(self):
+        ref = ConanFileReference.loads("lib1/1.0@conan/stable")
+        ref2 = ConanFileReference.loads("lib2/1.0@conan/stable")
+
+        client = TurboTestClient(servers={"default": TestServer()})
+        pref1 = client.create(ref)
+        client.upload_all(ref)
+
+        client.create(ref2, conanfile=GenConanfile().with_requirement(ref))
+        client.upload_all(ref2)
+
+        client.run("remove {} -p {} -f".format(pref1.ref, pref1.id))
+
+        os.makedirs(os.path.join(client.cache_folder, "data", ref.dir_repr(), "package", pref1.id))
+
+        client.run("install {}".format(ref2))
```

Comment:
What is or what is not happening? In this cases I would like at least to assert that something has downloaded (for example) and adding a comment about what we are testing.

Reply:
it was raising here before, but now command is successful
`client.run` already checks for conan install status code, but maybe I need to think on how to make it more explicit

Reply:
I added a check for an output (similar for other tests)